workflow-templates/jitpack_build_nodejs.yml

```
@@ -0,0 +1,39 @@
+name: Node.js Package
+
+on:
+  pull_request:
+    branches:
+      - master
```

Looks like you can use $default branch now so it won't matter if the default branch is master or main 👍🏼 

```suggestion
    branches:
      - $default-branch
      - refs/tags/*
```